tests/integration/targets/ios_banner/tests/cli/multiple-login-exec.yaml

```diff
@@ -29,12 +29,12 @@
   register: result
   cisco.ios.ios_banner:
     banner: exec
-    text: "this is my login banner\nthat as a multiline\nstring\n"
+    text: "this is my login banner\nthat as a multiline\nstring"
     state: present
-    provider: '{{ cli }}'
+    provider: "{{ cli }}"
 
 - debug:
-    msg: '{{ result }}'
+    msg: "{{ result }}"
 
```

Comment:
u need to add an integration tests with either `@` or `#` to have a verification at integration level as well